src/main/java/com/quantumsoft/qupathcloud/dao/spec/LocationsPathBuilder.java

```diff
@@ -19,14 +19,23 @@
 import static com.quantumsoft.qupathcloud.dao.Constants.PROJECTS;
 import static com.quantumsoft.qupathcloud.dao.Constants.STAGE;
 
-public class LocationsPathBuilder implements PathBuilder{
-    private QueryBuilder queryBuilder;
+/**
+ * LocationsPathBuilder to build a path to access all locations in the selected project.
```

Comment:
LocationsPathBuilder builds a path to access all locations in the selected project.